src/emqx_misc.erl

```diff
@@ -62,15 +62,11 @@ proc_stats(Pid) ->
 
 -define(DISABLED, 0).
 
+init_proc_mng_policy(undefined) -> ok;
 init_proc_mng_policy(Zone) ->
     #{max_heap_size := MaxHeapSizeInBytes}
         = ShutdownPolicy
-        = case Zone of
-              undefined ->
```

Comment:
then there is reason to keep this case clause

Reply:
Yes, of course.